Dockerfile

```diff
@@ -58,10 +58,20 @@ ENV PATH="/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:${PATH}
   XDG_CACHE_HOME=/home/linuxbrew/.cache
 WORKDIR /home/linuxbrew
 
-RUN mkdir -p \
+
+RUN --mount=type=cache,target=/tmp/homebrew-core,uid=1001,sharing=locked \
+    # Clone the homebre-core repo into /tmp/homebrew-core or pull latest changes if it exists
+    git clone https://github.com/homebrew/homebrew-core /tmp/homebrew-core || { cd /tmp/homebrew-core && git pull; } \
+    && mkdir -p /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core \
+    && cp -r /tmp/homebrew-core /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/
+
+
+RUN --mount=type=cache,target=/home/linuxbrew/.cache,uid=1001 \
+   --mount=type=cache,target=/home/linuxbrew/.bundle,uid=1001 \
+   mkdir -p \
   .linuxbrew/bin \
-  .linuxbrew/etc \
   .linuxbrew/include \
+  .linuxbrew/etc \
```

Comment:
Why move this?

Reply:
no reason, looks like a miss while I was probably testing this.

Reply:
Cool, please move it back.